app/assets/stylesheets/barchart.scss

```diff
@@ -0,0 +1,46 @@
+$gap: 5px;
+$totalRows: 101;
```

Comment:
I think the pattern we want here is to put these into a `_variables.scss` and import it in `application.scss`.